website/assets/scss/_blog.scss

```diff
@@ -0,0 +1,194 @@
+.blog{
+
+  &-rounded {
+    border-radius: .5 * $blog-gutter-xs;
+  }
+
+  &__pagination {
+    margin: $blog-gutter-md;
+
+    &>li>a {
+      &:focus {
+        outline: 0px;
+      }
+    }
+
+    &__active {
+      cursor: initial;
+      font-size: 1rem;
+      font-weight: bold;
+      color: $primary
+    }
+
+    &__page {
+      cursor:pointer;
+      font-size: .85rem;
+      padding: $blog-gutter-xs/5 $blog-gutter-xs;
+    }
+
+    &__previous {
+      &::before {
+        background: url('/images/page_prev@2x.svg') no-repeat center;
+      }
+    }
+
+    &__next {
+      &::before {
+        background: url('/images/page_next@2x.svg') no-repeat center;
+      }
+    }
+
+    &__button {
+      margin: 0 $blog-gutter-xs;
+      cursor:pointer;
```

Comment:
Doesn't prettier catch this kind of formatting mistakes like lacking space after colon here? Could you investigate?